src/resources/js/views/scripts.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * Makes sure we have all the required levels on the Tribe Object
+ *
+ * @since  TBD
+ *
+ * @type   {PlainObject}
+ */
+tribe.events = tribe.events || {};
+tribe.events.views = tribe.events.views || {};
+
+/**
+ * Configures Views Object in the Global Tribe variable
+ *
+ * @since  TBD
+ *
+ * @type   {PlainObject}
+ */
+tribe.events.views.script = {};
+
+/**
+ * Initializes in a Strict env the code that manages the Event Views
+ *
+ * @since  TBD
+ *
+ * @param  {PlainObject} $   jQuery
+ * @param  {PlainObject} _   Underscore.js
+ * @param  {PlainObject} obj tribe.events.views.scripts
+ *
+ * @return {void}
+ */
+( function( $, _, obj ) {
+	'use strict';
+	var $document = $( document );
+
+	/**
+	 * Selectors used for configuration and setup
+	 *
+	 * @since TBD
+	 *
+	 * @type {PlainObject}
+	 */
+	obj.selectors = {
+		tooltip: '.tribe-events-tooltip',
```

Comment:
can we use `data-js` attributes here rather than classes? this helps us separate what is used for styles and what we're targeting for js.

Reply:
I agree with the above.